tools/pre-upgrade.sh

```diff
@@ -28,3 +28,5 @@ sudo pip3 install -U "$_url"
 # Upgrade adapt-parser Python package
 _url="git+https://github.com/mycroftai/adapt#egg=adapt-parser"
 sudo pip3 install -U "$_url"
+
+sudo cp -r image/etc/ /
```

Comment:
We should probably do some proper permissions.

```bash
sudo chown -R root:root "image/etc"
sudo cp -r "image/etc" /
sudo chown -R $(id -u):$(id -g) "image/etc"
```

Reply:
:+1:, feel free to continue your PR and obsolete this one :)

Reply:
Nah, already deleted it. These were my changes exactly.

Reply:
Okay, updated this PR with those changes